### ISSUE TITLE
youtube integration?

### ISSUE THREAD

**Issue by JHeroGR** [giambaJ/jChat#35]
Any chance we can jChat integrated with youtube?

**Comment by SofiaEngvall** [comment]
Thant would be so great! <3

**Comment by JHeroGR** [comment]
Yeah I'm currently streaming on Trovo, Twitch and Facebook Gaming, so integrations to jChat for those features can be really awesome, including the YouTube chat integration
